cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,242 @@
-    insert_custom_fields(self)
-    salary_component(self)
-
-def insert_custom_fields(self):
-    # Check if there are any salary structure assignments
-    if len(self.salary_structure_assignment) > 0:
-        for assignment in self.salary_structure_assignment:
-            # Fetch existing custom field based on the field name and doctype
-            custom_field = frappe.db.get_list(
-                'Custom Field',
+@frappe.whitelist()
+def get_salary_component(data,component,custom_field):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_salary_component`

(ANN201)